pkg/apis/core/v1/defaults.go

```diff
@@ -201,6 +201,10 @@ func SetDefaults_Pod(obj *v1.Pod) {
 		defaultPodRequests(obj)
 	}
 
+	if utilfeature.DefaultFeatureGate.Enabled(features.PodLevelResources) && utilfeature.DefaultFeatureGate.Enabled(features.InPlacePodLevelResourcesVerticalScaling) {
```

Comment:
Per https://github.com/kubernetes/kubernetes/pull/133697 and https://github.com/kubernetes/kubernetes/pull/133912, I believe the correct / recommended way to do this now is to declare `InPlacePodLevelResourcesVerticalScaling` as having a dependency on `PodLevelResources`, and then here we only need to check `InPlacePodLevelResourcesVerticalScaling`.

@tallclair can confirm though 

Also here's the email ICYMI: https://groups.google.com/a/kubernetes.io/g/dev/c/OpUWIVR9DHY/m/1rFE2eaqAQAJ?utm_medium=email&utm_source=footer